src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -47,6 +49,9 @@ public Book findByAuthor(@PathVariable String author) {
 	@PostMapping
 	@ResponseStatus(HttpStatus.CREATED)
 	public Book create(@RequestBody Book book) {
+		if(bookRepository.findByUsers_Id(book.getUsers().getId()).isPresent()){
```

Comment:
So by this code if I am a user that has only one Book, and I want to add another book to my collection, this if sentence would prevent me to do that :thinking: